src/api/RebalancingAPI.ts

```diff
@@ -213,6 +213,25 @@ export class RebalancingAPI {
     return await this.rebalancingAuctionModule.bid(rebalancingSetTokenAddress, bidQuantity, txOpts);
   }
 
+  /**
+   * Allows user to bid and withdraw to wallet on a rebalance auction occuring on a Rebalancing Set Token
+   *
+   * @param  rebalancingSetTokenAddress     Address of the Rebalancing Set
+   * @param  bidQuantity                    Amount of currentSet the bidder wants to rebalance
+   * @param  txOpts                         Transaction options object conforming to `Tx` with signer, gas, and
+   *                                          gasPrice data
+   * @return                                Transaction hash
+   */
+  public async bidAndWithdrawAsync(
```

Comment:
The reason why we have this abstraction between api and wrapper is to allow for better interfaces on the api layer instead of always forwarding the call to the contract directly.

What I would prefer here is if there was only one method called `bidAsync` and then a separate parameter for `shouldWithdraw`. The method then routes to the correct contract depending on the `shouldWithdraw` flag. See `IssuanceAPI#redeemAsync` for examples.

Reply:
I would have it default to `true`?